Resources/Storage/StorageElement.py

```diff
@@ -397,39 +395,72 @@ def getOccupancy(self, unit='MB', **kwargs):
     filteredPlugins = self.__filterPlugins('getOccupancy')
     if not filteredPlugins:
       return S_ERROR(errno.EPROTONOSUPPORT, "No storage plugins to query the occupancy")
+
+    # Call occupancy plugin if requested
+    occupancyPlugin = self.options.get('OccupancyPlugin')
+    if occupancyPlugin:
+      res = ObjectLoader().loadObject(occupancyPlugin)
+      if not res['OK']:
+        return S_ERROR(errno.EPROTONOSUPPORT, 'Failed to load occupancy plugin %s' % occupancyPlugin)
+      log.verbose('Use occupancy plugin %s' % occupancyPlugin)
+      occupancyPlugin = res['Value'](self)
+      if occupancyPlugin.isUsable:
+        res = occupancyPlugin.getOccupancy()
+        if res['OK']:
+          occupancyDict = res['Value']
+          result = self.checkOccupancy(occupancyDict, unit)
+          if result['OK']:
+            return result['Value']
```

Comment:
Shouldn't this just `return result` ? If it should `return result['Value']` it deserves a comment

Reply:
Right, must be fixed to so...